spyder/plugins/switcher/plugin.py

```diff
@@ -108,11 +108,13 @@ class Switcher(SpyderPluginV2):
     def get_name():
         return _("Switcher")
 
-    def get_description(self):
-        return _("A multi-purpose switcher.")
+    @staticmethod
+    def get_description():
+        return _("Display and manage a multi-purpose switcher.")
```

Comment:
```suggestion
        return _("Quickly switch between files and other items.")
```

* Reframe this in terms of what the plugin actually does for users, like most of the other ones